lib/component.js

```diff
@@ -161,6 +161,21 @@ class Component extends WebComponent {
     }
   }
 
+  /**
+   * Returns the reference to the parent of the given name. An error is thrown out if no parent of
+   * such name is found.
+   * @param {string} tagName - name of the parent to search for.
+   * @example this.parentbyTagName('my-app');
```

Comment:
and maybe a better name would be `findPanelParentByTagName` - it's a mouthful, but since panel components _are_ actual DOM elements, it's ambiguous otherwise whether you're talking about DOM parents or Panel parents.